optuna/integration/__init__.py

```diff
@@ -41,6 +42,8 @@
     from optuna.integration.tensorflow import TensorFlowPruningHook  # NOQA
     from optuna.integration.tfkeras import TFKerasPruningCallback  # NOQA
     from optuna.integration.xgboost import XGBoostPruningCallback  # NOQA
+    if sys.version_info[0] == 3:
```

Comment:
I think that these two lines can be removed because this block is only executed if the major version of Python is 2.
(i.e., this condition never holds)